### PR TITLE
Fix ai bot workspace assign

### PR DESCRIPTION
**Pull request by kristina-fefelova**
**Pull Request Requirements:**

- Provide a brief description of the changeset.
- Include a screenshots if applicable
- Ensure that the changeset adheres to the [DCO guidelines](https://github.com/apps/dco).

<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NzU1M2Y2NDUzNmNiMTQyNWExMTY1ZjgiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIn0._L9J8YNNa27thekmvaRsLka_YOZpBWtis94hkQq1ke4">Huly&reg;: <b>UBERF-8677</b></a></sub>

### CODE REVIEW

File: services/ai-bot/pod-ai-bot/src/config.ts [hunk 1]
```diff
@@ -59,7 +60,8 @@ const config: Config = (() => {
     OpenAIBaseUrl: process.env.OPENAI_BASE_URL ?? '',
     MaxContentTokens: parseNumber(process.env.MAX_CONTENT_TOKENS) ?? 128 * 100,
     MaxHistoryRecords: parseNumber(process.env.MAX_HISTORY_RECORDS) ?? 500,
-    Port: parseNumber(process.env.PORT) ?? 4010
+    Port: parseNumber(process.env.PORT) ?? 4010,
+    ExternalTransactorURL: process.env.EXTERNAL_TRANSACTOR_URL === 'true'
```
Comment (haiodo) [inline services/ai-bot/pod-ai-bot/src/config.ts]: Why you need it?

Comment (kristina-fefelova) [reply services/ai-bot/pod-ai-bot/src/config.ts]: Removed

File: services/ai-bot/pod-ai-bot/src/config.ts [hunk 2]
```diff
@@ -35,6 +35,7 @@ interface Config {
   MaxContentTokens: number
   MaxHistoryRecords: number
   Port: number
+  ExternalTransactorURL: boolean
```
Comment (haiodo) [inline services/ai-bot/pod-ai-bot/src/config.ts]:
```suggestion
```

File: server/client/src/account.ts [hunk 3]
```diff
@@ -296,3 +304,52 @@ function getAccoutsUrlOrFail (): string {
...
+        method: 'assignWorkspace',
+        params: [token, email, workspace, role, personId, shouldReplaceAccount, undefined, personAccountId]
+      })
+    })
+  ).json()
+
+  return res.result as WorkspaceLoginInfo
+}
+
+export async function createAccount (
```
Comment (haiodo) [inline server/client/src/account.ts:334]: @lexiv0re we potentially should rate limit this API from account service?

**Review by haiodo — APPROVED** [review]

**Pull request merged**